src/WinXP/index.js

```diff
@@ -212,12 +234,41 @@ const reducer = (state, action = { type: '' }) => {
 function WinXP() {
   const [state, dispatch] = useReducer(reducer, initState);
 
+  const [isScreenSaverActive, setIsScreenSaverActive] = useState(false);
```

Comment:
We don't have time for this, but ideally as there's a whole bunch of Screen Saver logic here, it would have made sense to refactor this to a custom hook, say `useScreenSaver`, to keep the main file clean. Again, don't do it now :-)

Reply:
Got it.